.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Debug Build
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
do we want it on main or develop?

Reply:
Both.

Reply:
Main is usually updated from develop, so you would already have a build for it though